clang/include/clang/Sema/Sema.h

```diff
@@ -8126,7 +8143,7 @@ class Sema final : public SemaBase {
 
   /// The scope in which to find allocation functions.
   enum AllocationFunctionScope {
-    /// Only look for allocation functions in the global scope.
+    /// Only look for allocation functions in the global scope
```

Comment:
unrelated change?

Reply:
*looks shifty*

Nope, 100% absolutely critical for correctness